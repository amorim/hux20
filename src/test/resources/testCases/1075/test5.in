6 7
1 2 76
1 3 84
2 3 89
3 4 53
4 5 51
4 6 67
5 6 75