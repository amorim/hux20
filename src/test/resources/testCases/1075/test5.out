It's possible with distance 84